snuba/cli/consumer.py

```diff
@@ -135,13 +139,23 @@ def consumer(
     setup_logging(log_level)
     setup_sentry()
     logger.info("Consumer Starting")
-    storage_key = StorageKey(storage_name)
-    sentry_sdk.set_tag("storage", storage_name)
-
-    metrics_tags = {
-        "group": consumer_group,
-        "storage": storage_key.value,
+    storages = {
+        key: get_writable_storage(key)
+        for key in (StorageKey(name) for name in storage_names)
     }
+    storage_keys = [*storages.keys()]
+    sentry_sdk.set_tag("storage", ",".join(storage_names))
+
+    if len(storages) == 1:
+        metrics_tags = {
+            "group": consumer_group,
+            "storage": storage_keys[0].value,
+        }
+    else:
+        metrics_tags = {
+            "group": consumer_group,
+            "storage": "_".join([storage_keys[0].value, "m"]),
```

Comment:
I think this tag will end up not being very useful as you cannot search in Datadog by a substring of a tag (only prefix are allowed).
I think we should have a ticket to stop filtering our dashboards and alerts by storage and do it by consumer group as a consumer group is what identifies a consumer instead.
But then we could get into issues if we allowed multiple topics to be consumed by the same consumer on the same consumer group.

What do you think ?

Reply:
We talked about updating this to `",".join(storage_names)` or something. It is true that we normally assign different names to each group, though that is nothing more than a convention we use at Sentry and nothing else actually stops the same name from being used in multiple places. So I'd personally just leave the tag in there as it seems harmless.